templates/home/index.html.twig

```diff
@@ -1,8 +1,7 @@
 {% extends 'base.html.twig' %}
```

Comment:
so you have a "layout.html.twig", but this template extends "base.html.twig" directly?